### ISSUE TITLE
Custom element as a placeholder

### ISSUE THREAD

**Issue by janithmalshan** [TeamWertarbyte/material-ui-chip-input#225]
Is it possible to add ChipInput as a placeholder? 😊 Something similar to;

![image](https://user-images.githubusercontent.com/17025136/44506909-23410800-a6c6-11e8-9ef8-cc5a1ff6e311.png)

**Comment by leMaik** [comment]
I don't really know what you mean. Do you want to use a chip as placeholder?

**Comment by janithmalshan** [comment]
Exactly yes. :D
I want to use chip as a placeholder.

**Comment by rrundle** [comment]
Yeah this is def something custom. Since HTML only supports string placeholders you'd have to add a new element and absolute/fixed position it over the input. Then listen for a focus event and remove it if focused, then re-add it on blur if there is no value. Here's an example I found: http://jsfiddle.net/jrulle/tpnnsL64/
